src/components/react-json-schemaForm/TableField/Field/Field.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+
+
+const Field = props => {
+  const {
+    formData,
+    onChange,
+    registry: { fields: { SchemaField } },
+  } = props;
+
+  const [data, setData] = useState(formData);
+
+  const handleChange = value => {
+    setData(value);
+  };
```

Comment:
You need useCallback here 
```suggestion
  const handleChange = useCallback(value => {
    setData(value);
  }, []);
```

or you can also just keep the setData: 
```suggestion
  const handleChange = setData;
```
which is also working.

Reply:
Good point ! 👍 
I chose the third option ⬇️ 
```js
<SchemaField onChange={setData} />
```